app/views/brexit_checker/results.html.erb

```diff
@@ -74,7 +74,7 @@
       <%= render 'results_with_no_actions', criteria: @criteria %>
     <% end %>
 
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible govuk-section-break--bold">
```

Comment:
Could we replace this with a border-top on the subscribe section instead?

Reply:
done!